src/amqp10_client_session.erl

```diff
@@ -649,6 +657,44 @@ translate_terminus_durability(none) -> 0;
 translate_terminus_durability(configuration) -> 1;
 translate_terminus_durability(unsettled_state) -> 2.
 
+translate_filters(#{}) -> undefined;
+translate_filters(Filters) ->
```

Comment:
This will never match as the above clause does not match on the empty map - it matches on it _being_ a map and thus always passes. You need to use the `map_size` guard function to check for the empty map.